usertable.cpp

```diff
@@ -432,10 +432,9 @@ void UserTable::OnEvent(InotifyEvent& rEvt)
           oldpos = pos + 2;
         }
         else if (cs[px] == '&') {     // numeric mask
-          char* s;
-          asprintf(&s, "%u", (unsigned) rEvt.GetMask());
+          char s[11];
```

Comment:
> I think new throws an exception when the allocation fails.

True.

Well, obviously you have a good thought about it, so might as well just merge it :)